deck.gl__layers/index.d.ts

```diff
@@ -731,7 +894,7 @@ declare module '@deck.gl/layers/geojson-layer/geojson-layer' {
         getElevation?: ((d: D) => number) | number;
         getLineDashArray?: ((d: D) => [number, number]) | [number, number];
     }
-	export default class GeoJsonLayer<D> extends CompositeLayer<D> {
+	export default class GeoJsonLayer<D extends GeoJsonLayerData> extends CompositeLayer<D> {
```

Comment:
I think this is missing the generics as well as having the wrong type? For example you could have some GeoJson that is a:
```
FeatureCollection<Polygon, { elevation: number }>
```
but then trying to do:
```
new GeoJsonLayer({
  ...
  getElevation?: (feature: Feature<Polygon, { elevation: number }>) => feature.properties.elevation;
  ...
})
```
would fail because the typing `getElevation?: ((d: D) => number) | number;` plus `GeoJsonLayer<D extends GeoJsonLayerData>` puts:
```
D = FeatureCollection<Polygon, { elevation: number }>
```
which is incorrect.